lib/components/layouts/Sidebar/index.js

```diff
@@ -121,6 +105,28 @@ Sidebar.propTypes = {
    * The name of the app that is currently active
    */
   appName: PropTypes.string,
+  /**
+   * <div class="neeto-ui-tag neeto-ui-tag--size-small neeto-ui-tag--style-outline neeto-ui-tag--color-green mb-2">
+   * **Deprecated**
+   * </div>
```

Comment:
I think a danger style is more suitable for the `Deprecated` tag instead of green

<img width="879" alt="image" src="https://user-images.githubusercontent.com/22348659/187740366-331f9de0-1b7c-459e-9763-0da91aada997.png">